TYPO3.Neos/Resources/Public/JavaScript/Content/Navigate/AbstractNodeTree.js

```diff
@@ -855,9 +855,12 @@ define(
 				}
 				var that = this;
 				try {
-					sourceNode.move(targetNode, position === 'into' ? 'over' : position);
-					sourceNode.activate();
-					sourceNode.setLazyNodeStatus(this.statusCodes.loading);
+					// Don't do anything with sourceNode when moving across Document nodes
+					if (typeof sourceNode.parent.childList  !== 'undefined') {
```

Comment:
it fails since the sourceNode doesn't exist in the visible DOM, so checking for that would be better.. this can be done like this `if (document.contains(sourceNode.span)) {`